src/LocalProvider.js

```diff
@@ -163,4 +163,14 @@ export default class LocalProvider {
     call(request) {
         return request.execute(this._client);
     }
+
+    /**
+     * @returns {void}
+     */
+    close() {
+        this._client._network.close();
+        this._client._mirrorNetwork.close();
+        this._client._isShutdown = true;
+        clearTimeout(this._client._timer);
```

Comment:
Can we just call `this._client.close()` ?

Reply:
Yes, we can. For some reason it didn't work first time but now it's ok.